XRTK-Core/Packages/com.xrtk.core/Runtime/Extensions/InteractionSourceExtensions.cs

```diff
@@ -76,7 +76,9 @@ public static void StartHaptics(this InteractionSource interactionSource, float
                 }
             }, true);
 #elif UNITY_EDITOR_WIN
+#pragma warning disable 618
```

Comment:
Why disable warnings?

Reply:
Bc nothing will compile unless we do.

Too much work to convert it to native